packages/eslint-plugin/src/rules/no-unused-vars.ts

```diff
@@ -4,11 +4,33 @@ import {
   TSESTree,
 } from '@typescript-eslint/experimental-utils';
 import { PatternVisitor } from '@typescript-eslint/scope-manager';
-import baseRule from 'eslint/lib/rules/no-unused-vars';
+import { getNameLocationInGlobalDirectiveComment } from 'eslint/lib/rules/utils/ast-utils';
```

Comment:
this line breaks using eslint@5, which is supported: https://github.com/typescript-eslint/typescript-eslint/blob/1ef0d649886a47d7daeba7913f865476e596fd21/packages/eslint-plugin/package.json#L64

Reply:
see https://github.com/jest-community/eslint-plugin-jest/pull/732 for CI failure